v2/ecl/vna/v1/appliance_plans/requests.go

```diff
@@ -0,0 +1,111 @@
+package appliance_plans
+
+import (
+	"github.com/nttcom/eclcloud/v2"
+	"github.com/nttcom/eclcloud/v2/pagination"
+)
+
+// ListOpts allows the filtering and sorting of paginated collections through
+// the API. Filtering is achieved by passing in struct field values that map to
+// the Virtual Network Appliance Plan attributes you want to see returned.
+type ListOpts struct {
+	ID                        string `q:"id"`
+	Name                      string `q:"name"`
+	Description               string `q:"description"`
+	ApplianceType             string `q:"appliance_type"`
+	Version                   string `q:"version"`
+	Flavor                    string `q:"flavor"`
+	NumberOfInterfaces        int    `q:"number_of_interfaces"`
+	Enabled                   string `q:"enabled"`
+	MaxNumberOfAap            int    `q:"max_number_of_aap"`
+	Details                   string `q:"details"`
```

Comment:
```suggestion
	Details                   bool   `q:"details"`
```
APIではdatatypeがBooleanです。
https://sdpf.ntt.com/services/docs/firewall/api-references/rsts/APIList/source/list_virtual_network_appliance_plan.html#http-request-body